tour-of-heroes-api/index.js

```diff
@@ -14,14 +19,31 @@ const heroes = [
       { id: 20, name: 'Gypsy' },  
     ];
  
-app.get('/', function (req, res) {
-  res.send('Hello World')
-})
+// app.get('/', function (req, res) {
```

Comment:
can get rid of the commented out code, unless you want to turn it into a /test method with returned hello world so you know it worked